app/src/store/playground/types.ts

```diff
@@ -8,14 +8,28 @@ export type GenAIOperationType = "chat" | "text_completion";
  */
 export type PlaygroundInputMode = "manual" | "dataset";
 
+/**
+ * A tool call that invokes a function with JSON arguments
+ * @example { id: "1", function: { name: "getCurrentWeather", arguments: "{ \"city\": \"San Francisco\" }" } }
```

Comment:
small thing, you can do

```suggestion
 * @example 
 ```typescript
 { id: "1", function: { name: "getCurrentWeather", arguments: "{ \"city\": \"San Francisco\" }" } }
 ```
```
To make this look a little nicer maybe

Reply:
nice